lib/routes/sign.js

```diff
@@ -94,33 +94,37 @@ module.exports = function (log, isA, error, signer, db, domain) {
         var deviceId = sessionToken.deviceId ?
           sessionToken.deviceId.toString('hex') : null
 
-        return signer.sign(
-          {
-            email: uid + '@' + domain,
-            publicKey: publicKey,
-            domain: domain,
-            duration: duration,
-            generation: sessionToken.verifierSetAt,
-            lastAuthAt: sessionToken.lastAuthAt(),
-            verifiedEmail: sessionToken.email,
-            deviceId: deviceId
-          }
-        )
-        .then(
-          function(certResult) {
-            log.activityEvent(
-              'account.signed',
-              request,
+        return db.sessionTokenWithVerificationStatus(sessionToken.id)
+          .then( function(token) {
+            return signer.sign(
               {
-                uid: uid,
-                account_created_at: sessionToken.accountCreatedAt,
-                device_id: deviceId
+                email: uid + '@' + domain,
+                publicKey: publicKey,
+                domain: domain,
+                duration: duration,
+                generation: sessionToken.verifierSetAt,
+                lastAuthAt: sessionToken.lastAuthAt(),
+                verifiedEmail: sessionToken.email,
+                deviceId: deviceId,
+                sessionVerified: token.tokenVerificationId ? false : true
```

Comment:
You will need to change `./lib/signer.js` to actually include this value in the certificate; also the feature doc currently calls it `fxa-tokenVerified` rather than `fxa-sessionVerified`, and although I don't have a strong opinion that one is better than the other, we should keep them consistent.